bower.json

```diff
@@ -3,30 +3,29 @@
   "description": "Admin panel for the LANcie API",
   "main": "index.html",
   "dependencies": {
-    "iron-elements": "PolymerElements/iron-elements#^1.0.10",
-    "paper-elements": "PolymerElements/paper-elements#^1.0.7",
-    "gold-elements": "PolymerElements/gold-elements#^1.0.1",
-    "neon-elements": "PolymerElements/neon-elements#^1.0.0",
-    "polymer": "polymer/polymer#^1.7.0",
-    "wisvch-footer": "wisvch/wisvch-footer#^1.0.1",
-    "app-layout": "PolymerElements/app-layout#^0.10.4",
-    "app-route": "polymerelements/app-route#^0.9.2",
+    "polymer": "polymer/polymer#^2.0.0",
+    "app-layout": "PolymerElements/app-layout#^2.0.1",
+    "app-route": "polymerelements/app-route#^2.0.1",
+
     "iron-lazy-pages": "TimvdLippe/iron-lazy-pages#2.0-preview",
-    "lancie-login-card": "AreaFiftyLAN/lancie-login-card#1.0.0",
-    "lancie-ajax": "AreaFiftyLAN/lancie-ajax#1.0.1",
-    "vaadin-grid": "^1.2.1"
+    "lancie-login-card": "AreaFiftyLAN/lancie-login-card#2.0-preview",
+    "lancie-ajax": "AreaFiftyLAN/lancie-ajax#2.0-preview",
```

Comment:
We probably should publish these first as full 2.0.0 and then just switch to that, instead of relying on a branch